### DISCUSSION TITLE
Add CRT effects

### DISCUSSION THREAD

**Discussion by ajowens001** [shvchk/fallout-grub-theme#26] [Ideas]
Hello!  Thank you for your team's hard work on this theme; it is fantastic.

This isn't so much of an "Issue", but more of a request based on personal preference and nostalgia.

I was hoping to contact your team regarding the possibility of adding that CRT scrolling refresh, flicker, scanline effect that you might see on the actual PipBoy screen. 

Please reference below youtube video at time 0:56 for the short list of items I've mentioned.  The most important feature I really care about is the "Pulse" as mentioned in video; but it would be great if all effects were able to be implemented to your theme!
Listing of effects: https://youtu.be/xSDGclBT11A?t=56
Pulse!: https://youtu.be/xSDGclBT11A?t=80

The video description contains the times for each effect in question.

Imagine being able to have old world radio on this theme;  albeit the music would only play as long as you're choosing and booting, but still :)

Thanks for all you do and will do.

**Comment by shvchk** [comment]
GRUB does not support animated backgrounds or effects, unfortunately 😐
